format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt

```diff
@@ -59,17 +66,17 @@ internal object Otp {
         numDigits == null -> {
           return Err(IllegalArgumentException("Digits specifier has to be either 's' or numeric"))
         }
-        numDigits < 6 -> {
+        numDigits < MINIMUM_DIGIT -> {
           return Err(IllegalArgumentException("TOTP codes have to be at least 6 digits long"))
         }
-        numDigits > 10 -> {
+        numDigits > MAXIMUM_DIGIT -> {
           return Err(IllegalArgumentException("TOTP codes can be at most 10 digits long"))
```

Comment:
Use the constants defined above in the error messages.

Reply:
adjusted 👍